docs/README.FreeBSD.md

```diff
@@ -1,486 +1,237 @@
+
+You can install it with:
+```
+sudo pkg install distcc
+```
+
+### 3.1. Build missing dependencies
+See the general **[Linux build guide](README.Linux.md)** for reference.
+
+**[back to top](#table-of-contents)** | **[back to section top](#3-install-the-required-packages)**
+
+## 4. Build Kodi
+### 4.1. Configure build
+If you get a `Could NOT find...` error message during CMake configuration step, take a note of the missing dependencies and either install them from repositories (if available) or **[build the missing dependencies manually](#31-build-missing-dependencies)**.
+
+Create an out-of-source build directory:
+```
+mkdir $HOME/kodi-build
+```
+
+Change to build directory:
+```
+cd $HOME/kodi-build
+```
+
+Configure build:
+```
+cmake ../kodi -DCMAKE_INSTALL_PREFIX=/usr/local
+```
 
-.1  $ git clone -b <branch> git://github.com/xbmc/xbmc.git kodi
-
------------------------------------------------------------------------------
-3. Installing the required libraries and headers
------------------------------------------------------------------------------
-
-You will then need the required libraries. The following is the list of packages
-that are used to build Kodi packages on Debian/Ubuntu (with all supported
-external libraries enabled).
-
-$ pkg install taglib gstreamer1-vaapi hal libcapn \
-enca gawk gperf cmake zip nasm swig30 libssh openjdk8 libtool gettext-tools \
-gmake pkgconf rapidjson mesa-libs doxygen glproto dri2proto dri3proto libass \
-flac libcdio curl dbus fontconfig freetype2 fribidi \
-libgcrypt gmp libgpg-error gnutls libidn libinotify lzo2 \
-libogg sqlite3 tiff tinyxml e2fsprogs-libuuid git libvorbis libxslt libplist \
-shairplay avahi-app libcec libbluray samba46 libnfs librtmp libva libvdpau \
-jpeg-turbo glew xrandr libedit inputproto giflib m4 encodings \
-font-util mysql57-client xf86vidmodeproto python2 p8-platform libbdplus \
-libaacs libudev-devd sndio ccache xorg-server binutils libmicrohttpd \
-xorg-server xf86-input-mouse xf86-input-keyboard lirc libfmt autoconf automake
-
------------------------------------------------------------------------------
-4. How to compile
------------------------------------------------------------------------------
-Cmake build instructions V18.0 Leia and higher
+### 4.2. Build
+```
+cmake --build . -- VERBOSE=1 -j$(sysctl hw.ncpu | awk '{print $2}')
+```
+**TIP:** By adding `-j<number>` to the make command, you can choose how many concurrent jobs will be used and expedite the build process. It is recommended to use `-j$(sysctl hw.ncpu | awk '{print $2}')` to compile on all available processor cores.
+
+After the build process completes successfully you can test your shiny new Kodi build while in the build directory:
+```
+./kodi-x11
+```
```

Comment:
no wayland (and maybe also gbm) instructions?

Reply:
Will be tested while reviewing guides.